src/main/flight/pid.c

```diff
@@ -292,7 +293,18 @@ void pidUpdateTpaFactor(float throttle)
     } else {
         tpaRate = pidRuntime.tpaLowMultiplier * (pidRuntime.tpaLowBreakpoint - throttle);
     }
-    pidRuntime.tpaFactor = 1.0f - tpaRate;
+
+    float tpaFactor = 1.0f - tpaRate;
+    DEBUG_SET(DEBUG_TPA, 0, lrintf(tpaFactor * 1000));
+
+#ifdef USE_WING
+    if (isFixedWing()) {
+        tpaFactor = pt2FilterApply(&pidRuntime.tpaLpf, tpaFactor);
+    }    
+#endif
+
+    DEBUG_SET(DEBUG_TPA, 1, lrintf(tpaFactor * 1000));
+    pidRuntime.tpaFactor = tpaFactor;
```

Comment:
This is a bit pedantic but one final suggestion would be to manipulate `pidRuntime.tpaFactor` directly,
1) to save the memory allocation for `tpaFactor`
2) to save a copy instruction for copying `tpaFactor` into `pidRuntime.tpaFactor`

This is slightly faster but I don't really care too much 😀

```suggestion
    pidRuntime.tpaFactor = 1.0f - tpaRate;
    DEBUG_SET(DEBUG_TPA, 0, lrintf(pidRuntime.tpaFactor * 1000.0f));

#ifdef USE_WING
    if (isFixedWing()) {
        pidRuntime.tpaFactor = pt2FilterApply(&pidRuntime.tpaLpf, pidRuntime.tpaFactor);
    }    
#endif

    DEBUG_SET(DEBUG_TPA, 1, lrintf(pidRuntime.tpaFactor * 1000.0f));
```

Reply:
See https://github.com/betaflight/betaflight/pull/13662#discussion_r1623250432

Reply:
yeah this is how i made it initially, but ledvinap suggested a local variable is a better style. Considering compiler will optimize it anyways.

Reply:
@KarateBrot : local variables are cheap and in most cases placed in registers. So they're should be no actual overhead. 
I requested this change to make sure that pidRuntime.tpaFactor is always consistent. Your suggestion allows unfiltered value to be stored in pidRuntime.tpaFactor for few instructions, making it inconsistent. It is not a problem with current concurrency implementation, but it may cause problems eventually